platforms/r3-corda/configuration/molecule/doorman/converge.yml

```diff
@@ -0,0 +1,91 @@
+---
+- name: Converge
+  hosts: localhost
+  connection: local
+  vars:
+    privilege_escalate: true            #escalate root priviledge for default test
+    install_os: "linux"                 #Default to linux OS
+    install_arch:  "amd64"              #Default to amd64 architecture
+    bin_install_dir:  "~/bin"
+    kubernetes: 
+      config_file: "/tmp/molecule/kind-default/kubeconfig"
+      context: "kind"
+    network:
+      type: "corda"
+      docker:
+        url: "test_url"
+        username: "test_username"
+        password: "test_password"
+      orderers:
+      - orderer:
+        type: doorman
+        uri: https://doormankd.rc.dev.aws.blockchaincloudpoc-develop.com:8443
+        certificate: /home/deepak/BFA/blockchain-automation-framework/platforms/r3-corda/configuration/molecule/doorman/build/corda/testdoorman/tls/ambassador.crt        
```

Comment:
Path should be from molecule test dir. And you have to checkin the sample certificate if needed